plugins/htpasswd/scripts/github-release.sh

```diff
@@ -0,0 +1,8 @@
+# Get the last tag from GitHub
+lastTag=$(git describe --tags $(git rev-list --tags --max-count=1))
+
+changelog=$(git show $GITHUB_SHA --unified=0 CHANGELOG.md | tail +12 | sed -e 's/^\+//')
```

Comment:
I think the `scripts` folder is not needed because the scripts contained are based on a single project repository behaviour

Reply:
All GitHub hidden folder should be removed.